input/fsh/package-operation.fsh

```diff
@@ -92,7 +92,7 @@ TODO: More documentation about the operation, including inline examples:
   * min = 0
   * max = "*"
   * use = #in
-  * type = #string
+  * type = #code
```

Comment:
I feel `code` is a better type here since it's a constrained list of possible values? (accordingly I guess we'll need a ValueSet...)

Reply:
This has been submitted as a ballot comment - Jira#:[FHIR-38836](https://jira.hl7.org/browse/FHIR-38836). We should make the change as part of a dedicated PR for resolution of that issue and it probably ought to include the proposed binding and relevant value set.